src/UglyToad.PdfPig.Tokens/NumericToken.cs

```diff
@@ -155,13 +156,32 @@ public class NumericToken : IDataToken<decimal>
         /// <summary>
         /// The value of this number as a <see langword="double"/>.
         /// </summary>
-        public double Double => (double)Data;
+        public double Double => Data;
 
         /// <summary>
         /// Create a <see cref="NumericToken"/>.
         /// </summary>
         /// <param name="value">The number to represent.</param>
+        //[Obsolete("Use double constructor instead.")]
```

Comment:
I can't run the tests if I mark this method obsolete... Maybe there are some rules somewhere that prevent that...

Reply:
Is it still used after this changeset or could it be removed in a follow-up PR? I think the next version bump will be quite breaking anyway so in for a penny in for a pound, just remove it if unneeded internally

Reply:
agreed yes, I'll remove that in a follow up PR